cfg/validate.go

```diff
@@ -55,10 +52,10 @@ func isValidFileCacheConfig(config *FileCacheConfig) error {
 		if config.MaxParallelDownloads == 0 {
 			return fmt.Errorf("the value of max-parallel-downloads for file-cache must not be 0 when enable-parallel-downloads is true")
```

Comment:
Revert the removed line and use it here
```suggestion
			return fmt.Errorf(MaxParallelDownloadsCantBeZeroError)
```

Reply:
Done